### ISSUE TITLE
Bugged quest: Hunt da' Stunies!

### ISSUE THREAD

**Issue by Bahlthazar** [WarEmu/WarBugs#17329]
Every time I try to complete the quest it tells me "Cannot Salvage. Inventory overflowing. Open backpack to remove overflow" 
I've tried relogging, reloaded, and restarting the game entirely. Nothing has worked.
![Screenshot (810)](https://user-images.githubusercontent.com/58861102/87744764-8734b400-c7a9-11ea-8afb-a2dcce7b9775.png)

**Comment by Kaelang** [comment]
Do you have a full inventory?

**Comment by Bahlthazar** [comment]
> Do you have a full inventory?

No. I cleared most of my bags just to see if that was it. Still didn't work.
